src/OpcUaPiXtendServer/Config/OpcUaPiXtendServer.xml

```diff
@@ -53,7 +53,15 @@
             <Address>0x03</Address>
             <USBDevice Device="/dev/ttyUSB0"/>
 	    </Module>
-	
     </Modules>
 
+    <Objects>
```

Comment:
The example object is not displayed in the OPC UA information model. 

This also applies if the enable variable contains the value 1.